src/components/BarChart.vue

```diff
@@ -40,12 +53,17 @@ a {
 
 .bar-chart {
   display: flex;
-  flex-direction: row;  
+  flex-direction: row;
 
   width: 500px;
   height: 500px;
-  background-color: grey;
-  
+  background-image: linear-gradient(#fafffc, #e3e8f8);
+  padding: 20px;
+  margin: 30px;
+  border-radius: 5px;
+  box-shadow: 3px 3px 20px #e9e9e9;

```

Comment:
I tend to use values that are multiples of 16 (4, 8, 12, 16, 20 etc), they generally scale better. 16px is also the default font size of the browsers so using sizes based of that usually makes for somewhat more optically pleasing UI's

Reply:
ok that's very interesting, thanks!